From: Ilya Dryomov <idryomov@gmail.com>
Date: Mon, 18 Jan 2016 16:53:31 +0100
Subject: [PATCH] libceph: remove outdated comment
Git-commit: 7e01726a6853e032536ed7e75c1e1232872ff318
Patch-mainline: v4.5
References: Fate#318586


MClientMount{,Ack} are long gone.  The receipt of bare monmap doesn't
actually indicate a mount success as we are yet to authenticate at that
point in time.

Signed-off-by: Ilya Dryomov <idryomov@gmail.com>
Acked-by: NeilBrown <neilb@suse.com>

---
 net/ceph/mon_client.c |    4 ----
 1 file changed, 4 deletions(-)

--- a/net/ceph/mon_client.c
+++ b/net/ceph/mon_client.c
@@ -364,10 +364,6 @@ static bool have_debugfs_info(struct cep
 	return monc->client->have_fsid && monc->auth->global_id > 0;
 }
 
-/*
- * The monitor responds with mount ack indicate mount success.  The
- * included client ticket allows the client to talk to MDSs and OSDs.
- */
 static void ceph_monc_handle_map(struct ceph_mon_client *monc,
 				 struct ceph_msg *msg)
 {
